### ISSUE TITLE
/model/predict endpoint fails on Red Hat openShift due to permission error

### ISSUE THREAD

**Issue by ptitzler** [IBM/MAX-Sports-Video-Classifier#29]
```

[2019-08-07 01:00:04,681] ERROR in app: Exception on /model/predict [POST]
--
  | Traceback (most recent call last):
  | File "/opt/conda/lib/python3.6/site-packages/flask/app.py", line 1832, in full_dispatch_request
  | rv = self.dispatch_request()
  | File "/opt/conda/lib/python3.6/site-packages/flask/app.py", line 1818, in dispatch_request
  | return self.view_functions[rule.endpoint](**req.view_args)
  | File "/opt/conda/lib/python3.6/site-packages/flask_restplus/api.py", line 319, in wrapper
  | resp = resource(*args, **kwargs)
  | File "/opt/conda/lib/python3.6/site-packages/flask/views.py", line 88, in view
  | return self.dispatch_request(*args, **kwargs)
  | File "/opt/conda/lib/python3.6/site-packages/flask_restplus/resource.py", line 44, in dispatch_request
  | resp = meth(*args, **kwargs)
  | File "/opt/conda/lib/python3.6/site-packages/flask_restplus/marshalling.py", line 136, in wrapper
  | resp = f(*args, **kwargs)
  | File "/workspace/api/predict.py", line 61, in post
  | video_data.save(filepath)
  | File "/opt/conda/lib/python3.6/site-packages/werkzeug/datastructures.py", line 2799, in save
  | dst = open(dst, "wb")
  | PermissionError: [Errno 13] Permission denied: './assets/baseball.mp4'


```

**Comment by djalova** [comment]
@ptitzler How was this tested? The `baseball.mp4` file shouldn't be in the `assets` directory anymore.

**Comment by ptitzler** [comment]
The specific code is trying to save the uploaded file in this directory. I just happened to use the sample video that used to be in that folder.

**Comment by djalova** [comment]
Resolved by #32
